.github/workflows/ci-polaris-react.yml

```
@@ -10,36 +10,6 @@ on:
       - 'polaris-react/**'
```

```suggestion
      - 'polaris-react/**'
```
Should this run without the filter? The file could also be renamed?!

---

Probably so, since a change in other packages might cause a change in the Viz and A11y tests. Good callout

---

At this point, where there's no filtering here and the `on` logic is the same as ci.yml, could these be jobs in the ci.yml file?

---

Decided to keep the filter and CI separate since jobs can't be filtered based on path. Only full workflows can be triggered conditionally by path.